broker-node/app/Clients/BrokerNode.php

```diff
@@ -27,10 +28,13 @@ class BrokerNode
 
     public static $ChunkEventsRecord = null;
 
-    private static function initEventRecord()
+    private static $SegmentStarted = null;
```

Comment:
We can add `AltThree\Segment\SegmentServiceProvider::class,` on line `171` in `broker-node/config/app.php`

Sorry, forgot to initialize it.

Reply:
Nvm, I see you already did it. The `ENV` variable just wasn't set up, which is why you might have been seeing errors.